scripts/create-backup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
I think we should not modify shebangs of the script since `writeScriptBin` does not alter it and `#!/bin/sh` works.

Reply:
Done in https://github.com/osrd-project/osrd/compare/3dbf7d69f53eab1915cd691738cb64fb70bc1edd..87aca65215c352c49f677f6a664beebbde1564b1